sdcm/nemesis.py

```diff
@@ -453,10 +431,10 @@ def set_target_node(self, dc_idx: Optional[int] = None, rack: Optional[int] = No
                 self.target_node = random.choice(nodes)
 
             if current_disruption:
-                self.target_node.running_nemesis = current_disruption
+                set_running_nemesis(self.target_node, current_disruption, self.disruptive)
```

Comment:
`_get_target_nodes` anyway works the old way - never selecting any node that runs any nemesis, regardless of descriptiveness.  So selecting node during nemesis startup will work the old way. 
I'm not saying it's wrong, just noting it.